module2/web/src/main/java/com/epam/esm/web/config/Initializer.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.web.config;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.ContextLoaderListener;
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletRegistration;
+
+public class Initializer implements WebApplicationInitializer {
+
+    @Override
+    public void onStartup(ServletContext servletContext) {
+        AnnotationConfigWebApplicationContext context = new AnnotationConfigWebApplicationContext();
+        context.register(WebConfig.class);
+        context.setServletContext(servletContext);
+        servletContext.addListener(new ContextLoaderListener(context));
+        servletContext.setInitParameter("spring.profiles.active", "prod");
```

Comment:
It's bad practice to set profile in code

Reply:
fixed